src/Farmer/Builders/Builders.WebApp.fs

```diff
@@ -72,6 +72,17 @@ type SecretStore =
     | AppService
     | KeyVault of ResourceRef<WebAppConfig>
 
+//
+// I really want to call this a
+// - SiteExtension (annoys me that it collides with type with same name from module SiteExtension) or
+// - SiteExtensionConfiguration (doesn't seem to fit with naming scheme around these parts)
+//
+type Extension = {
+    Name : string
+    Location : Location option // Defaults to main location
```

Comment:
Will we ever be in a situation where the Location is different from the web app?

Reply:
That's a question for an Azure expert. All I know is, you **can** specify a different location. I agree that we can go with a very simple implementation of `use_extension` for now, and simplify the data types accordingly, knowing that if someone logs an issue (asking for specific version or location specification) that we can then go and make a builder. 
I'll remove references to Location for now and simplify

Reply:
Agreed, lets remove Location for now.